book/src/leader-validator-transition.md

```diff
@@ -0,0 +1,85 @@
+# Leader-to-Validator Transition
+
+A fullnode typically operates as a validator. If, however, a staker delegates
+its stake to a fullnode, it will occasionally be selected as a *slot leader*.
```

Comment:
"it will" => "it may".  I don't think we make any guarantees right?  I stake my 1 token, I'll still never be selected as leader.

Reply:
Over a sufficiently massive number of leader schedules, it will get its day in the sun.